src/web/metrics.rs

```diff
@@ -257,9 +288,8 @@ mod tests {
     };
 
     static ROUTES_VISITED: AtomicUsize = AtomicUsize::new(0);
-    lazy_static::lazy_static! {
-        static ref RESPONSE_TIMES: Mutex<HashMap<String, usize>> = Mutex::new(HashMap::new());
-    }
+    static RESPONSE_TIMES: Lazy<Mutex<HashMap<String, usize>>> =
+        Lazy::new(|| Mutex::new(HashMap::new()));
```

Comment:
Seems a shame to put a mutex inside a mutex ... don't we have to worry about deadlocks for something like that?

Reply:
(This shouldn't block the PR though since it was a pre-existing issue)

Reply:
These can't deadlock since the writes don't depend on each other

Reply:
Ah right, because Lazy only uses a mutex before it's been initialized, and the inner mutex doesn't exist until initialization finished. Got it.
```suggestion
        // can't deadlock: Lazy only uses a mutex before it's been initialized, and the inner mutex doesn't exist until initialization finished
        Lazy::new(|| Mutex::new(HashMap::new()));
```

Reply:
I’m a little confused as to why a comment is needed since the two locks don’t interact with each other . `Lazy` has the semantics of `lazy_static` so anyone familiar with either of the libs wouldn’t even second guess this since it’s a super common use case (with mutexes not being const constructible)